src/Tizen.MachineLearning.Nnstreamer/Tizen.MachineLearning.Nnstreamer/Commons.cs

```diff
@@ -0,0 +1,230 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.IO;
+
+namespace Tizen.MachineLearning.Nnstreamer
+{
+    /// <summary>
+    /// Enumeration for pipeline state.
+    /// </summary>
+    /// <since_tizen> 5.5 </since_tizen>
+    public enum PipelineState
+    {
+        /// <summary>
+        /// Unknown state. Maybe not constructed?
+        /// </summary>
+        Unknown = 0,
```

Comment:
I think, Unknown is not good. but i can't recommend alternative because i don't known why need this enum

Reply:
ditto.